zoo-css-challenge/style.css

```
@@ -184,12 +235,19 @@ footer {
   color: #ff3f00;
 }
 
+.membership p {
+  max-width: 700px;
+  margin-left: 100px;
```

```suggestion
  margin: auto;
```
Alternatively, we can use `margin: auto` which centers the element by distributing the margins around equally